cmd/layotto/main.go

```diff
@@ -431,6 +432,11 @@ func NewRuntimeGrpcServer(data json.RawMessage, opts ...grpc.ServerOption) (mgrp
 			secretstores_loader.NewFactory("local.env", func() secretstores.SecretStore {
 				return secretstore_env.NewEnvSecretStore(loggerForDaprComp)
 			}),
+		),
+		runtime.WithSecretWrapperFactory(
+			secret.NewWrapperFactory("local.file", func() secret.Wrapper {
+				return secret.NewLocalFileWrapper()
```

Comment:
Sorry u misunderstood it 😢 
I think we can write a wrapper to **wrap the existing local.file component** .
I submit a PR to your branch to illustrate the solution. See https://github.com/MichaelDeSteven/layotto/pull/2
Please review and merge it :)

Reply:
> Sorry u misunderstood it 😢 I think we can write a wrapper to **wrap the existing local.file component** . I submit a PR to your branch to illustrate the solution. See [MichaelDeSteven#2](https://github.com/MichaelDeSteven/layotto/pull/2) Please review and merge it :)

I am doing this, hold on plz.